src/bin/Makefile

```diff
@@ -7,7 +7,7 @@ BINFILES = align-equal align-equal-compiled acc-tree-stats \
         show-alignments compile-questions cluster-phones \
         compute-wer compute-wer-bootci make-h-transducer \
```

Comment:
@xiaohui-zhang, if you want pre-merged MR from GitHub, you can do
```
git fetch origin +refs/pull/579/merge:
git checkout FETCH_HEAD
```
(`origin` here stands for the remote name of the main Kaldi repo).